### PR TITLE
Update MetaDataContexts.java

### PR DESCRIPTION
**Pull request by DivvSaxena**
Fixes 

Changes proposed in this pull request:
  -Unify metadata naming
  -
  -

---

Before committing this PR, I'm sure that I have checked the following options:
- [ ] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [ ] I have self-reviewed the commit code.
- [ ] I have (or in comment I request) added corresponding labels for the pull request.
- [ ] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

File: infra/util/src/main/java/org/apache/shardingsphere/infra/util/spi/aware/SPIMetadataAware.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
 import java.util.Collection;
 
 /**
- * SPI metadata aware.
+ * SPI metadata aware.
```
Comment (RaigorJiang) [inline infra/util/src/main/java/org/apache/shardingsphere/infra/util/spi/aware/SPIMetadataAware.java:23]: Just added a space here?

Comment (DivvSaxena) [reply infra/util/src/main/java/org/apache/shardingsphere/infra/util/spi/aware/SPIMetadataAware.java:23]: yes , I think while reviewing so many files I added that by mistake

Comment (RaigorJiang) [reply infra/util/src/main/java/org/apache/shardingsphere/infra/util/spi/aware/SPIMetadataAware.java:23]: This whitespace is unnecessary, so can we close this PR?

Comment (DivvSaxena) [reply infra/util/src/main/java/org/apache/shardingsphere/infra/util/spi/aware/SPIMetadataAware.java:23]: Okay

**Pull request closed without merging**